.repo-metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "nodejs-proto-files",
+  "name_pretty": "Google APIs Proto Files",
+  "release_level": "ga",
+  "language": "nodejs",
+  "product_documentation": "https://cloud.google.com/nodejs/docs/reference/common/latest/",
```

Comment:
this is a 404, I wonder if perhaps https://cloud.google.com/nodejs/docs/reference/libraries would be an appropriate URL?